app/positions/components/FromAndToMarkets.tsx

```
@@ -0,0 +1,202 @@
+                <th className="px-4 py-2 text-left">LLTV</th>
+                <th className="px-4 py-2 text-left">APY</th>
+                <th className="px-4 py-2 text-left">Supplied Amount</th>
+              </tr>
+            </thead>
+            <tbody className="text-sm">
+              {paginatedFromMarkets.map((marketPosition) => (
+                <tr
+                  key={marketPosition.market.uniqueKey}
+                  onClick={() => onFromMarketSelect(marketPosition.market.uniqueKey)}
+                  className="cursor-pointer border-b border-gray-200 hover:bg-gray-50 dark:border-gray-700 dark:hover:bg-gray-800"
+                >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)